guides/common/modules/proc_acd_installation.adoc

```diff
@@ -45,6 +45,31 @@ endif::[]
 # {foreman-maintain} service restart
 ----
 
+[id="{context}_installation_{SmartProxyServer}"]
+= Installing ACD on {SmartProxyServer}
+
+You need to install two packages to use ACD on a {SmartProxy}.
+
+.Procedure
+ifdef::foreman-el,foreman-deb,katello,satellite[]
+. Create a new yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {SmartProxyServer}:
```

Comment:
Create implies new, so no need to say new! 
```suggestion
. Create a yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {SmartProxyServer}:
```

Reply:
Fixed.